.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    if: "! contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
```

Comment:
can you please add this https://github.com/americanexpress/one-service-worker/pull/19

Reply:
``` with:
        persist-credentials: false```